patterns/1-initial/explicit-innersource-principles.md

```diff
@@ -157,7 +157,35 @@ projects.
 
 ## Known Instances
 
-* Europace AG - see also [Europace InnerSource Prinzipien](https://tech.europace.de/post/europace-inner-source-prinzipien/) (German)
+### Europace AG
+
+- see also [Europace InnerSource Prinzipien](https://tech.europace.de/post/europace-inner-source-prinzipien/) (German)
+
+### Robert Bosch GmbH
+
+Fostering collaboration, learning and innovation is the main focus of the Bosch
+InnerSource initiative (BIOS). To that end, they applied the following
+principles:
+
+- **Openness**: We lower the barriers for entry into BIOS communities as much
+  as we can.
+- **Transparency**: We are radically transparent and share our work products,
+  our communication, and our decision making with all associates in the company.
+- **Voluntariness**: The decision to join and contribute to a BIOS community is
+  left to each associate. Associates should work within BIOS because they are
+  intrinsically motivated, not because their manager told them so.
+- **Self-Determination**: BIOS-Communities are free to choose what to work on,
+  when they work and what tools and processes they use to work.
+- **Meritocracy**: Power is vested in BIOS project members based ontheir merits,
+  that is, based on the quality and quantity of their contributions.
```

Comment:
Could you remove the linebreaks within the bullet points? Otherwise this won't get displayed correctly in the book.